EngineLayer/Indexing/IndexingEngine.cs

```diff
@@ -155,14 +155,9 @@ protected override MetaMorpheusEngineResults RunSpecific()
             if (GeneratePrecursorIndex)
             {
                 // create precursor index
-                try
-                {
-                    precursorIndex = new List<int>[(int)Math.Ceiling(MaxFragmentSize) * FragmentBinsPerDalton + 1];
-                }
-                catch (OutOfMemoryException)
-                {
-                    throw new MetaMorpheusException("Max precursor mass too large for indexing engine; try \"Classic Search\" mode, or make the maximum fragment mass smaller");
-                }
+
+                precursorIndex = new List<int>[(int)Math.Ceiling(MaxFragmentSize) * FragmentBinsPerDalton + 1];
```

Comment:
still need to catch exception

Reply:
This won't throw an exception. If the size is too large, it will have thrown an exception for the fragments (earlier in the code) and will not run this statement.